capellambse/model/crosslayer/__init__.py

```diff
@@ -6,7 +6,7 @@
 .. diagram:: [CDB] BaseLayer
 """
 from .. import common as c
-from . import cs, information
+from . import capellacommon, capellacore, cs, fa, information, interaction
```

Comment:
This is for being able to import them from the crosslayer for type-annotations.

Reply:
I still don't get what the issue is. You can just say `from capellambse.model.crosslayer import capellacommon`, regardless of whether or not this import exists here, can you not? `cs` and `information` are necessary because they're used in code that lives in this specific file, but the new imports still look redundant to me.